Source/Dafny/Compilers/Compiler-cpp.cs

```diff
@@ -2407,7 +2407,7 @@ protected class ClassWriter : IClassWriter {
     public override bool RunTargetProgram(string dafnyProgramName, string targetProgramText, string/*?*/ callToMain, string targetFilename, ReadOnlyCollection<string> otherFileNames,
       object compilationResult, TextWriter outputWriter) {
       var exeName = ComputeExeName(targetFilename);
-      var psi = new ProcessStartInfo(exeName) {
+      var psi = new ProcessStartInfo($"{Path.GetDirectoryName(Path.GetFullPath(targetFilename))}/{exeName}") {
```

Comment:
Rather than the changes on L2384 and L2410, I suggest changing the implementation of `ComputeExeName` to do what you're doing here. That is, I suggest implementing `ComputeExeName` as:

``` c#
      var dir = Path.GetDirectoryName(Path.GetFullPath(targetFilename));
      var filename = Path.GetFileNameWithoutExtension(targetFilename) + ".exe";
      return Path.Join(dir, filename);
```